docs/root/operations/admin.rst

```diff
@@ -174,3 +174,30 @@ The fields are:
   .. http:get:: /runtime?format=json
 
   Outputs /runtime in JSON format. This can be used for programmatic access of runtime values.
+
+.. http:get:: /hystrix_event_stream
+
+  Start streaming
+  `event stream <https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events>`_
+  to Hystrix dashboard. Detalied description of Hystrix and its dashboard can be found
+  `here <https://github.com/Netflix-Skunkworks/hystrix-dashboard>`_.
+
+  The dashboard should set the stream source to this admin endpoint.
+
+  Notes on the usage of Hystrix dashboard to visualize Envoy statistics:
+     Not all the data presented in the Hystrix dashboard is relevant in Envoy.
+
+    * Success, Failure, Timeout, Error rate, Number of host, are shown in the dashboard.
+    * Hystrix's "Short circuit" is similar to Envoy's outlier detection, in that it is triggered by
```

Comment:
Generally we do `*Short circuit*` rather than quoting in Envoy docs to call out distinguished terms, but that's mostly done for headers. I don't feel that strongly, just pointing it out.